gui/WorldView.cxx

```diff
@@ -1035,13 +1087,40 @@ void WorldView::setMesh(vtkSmartPointer<vtkPolyData> mesh)
 
   // Add this actor to the renderer
   d->renderer->AddActor(d->volumeActor);
-
-  if (mesh->GetPointData()->HasArray("RGB"))
+  bool rgb = false, rgba = false;
+  const char* arrayName = nullptr;
+  const char* rgbName = "RGB";
+  const char* rgbaName = "RGBA";
```

Comment:
```suggestion
  char const* arrayName = nullptr;
  constexpr auto rgbName = "RGB";
  constexpr auto rgbaName = "RGBA";
```

Please ***always*** write `T const`, never `const T`. I believe most Vision projects are following this these days, but definitely KWIVER, Krest and TeleSculptor are.

Reply:
Sounds good.